dags/openshift_nightlies/models/release.py

```diff
@@ -44,7 +44,7 @@ def _generate_cluster_name(self):
         if self.platform == 'rosa' or self.platform == 'rogcp' or self.platform == 'hypershift':
             #Only 15 chars are allowed
             cluster_version = str(self.version).replace(".","")
-            return "perf-"+cluster_version+"-"+md5(cluster_name.encode("ascii")).hexdigest()[:4]
+            return "perf-"+md5(cluster_name.encode("ascii")).hexdigest()[:3]
```

Comment:
there are character limitation set by `ocm`, we wanted to suffix a set of random characters per dag so using a logic to get random string based on the cluster_name.  We can't use`uuid` as it would set unique name on every run which we don't need.